test/forms/mappings/ConstraintsSpec.scala

```diff
@@ -116,28 +116,4 @@ class ConstraintsSpec extends WordSpec with MustMatchers with Constraints {
       result mustEqual Invalid("error.length", 10)
     }
   }
-
-  "inRange" must {
```

Comment:
Any reason this has gone? I think this was added as it wasn't covered by scoverage.